_includes/manuals/1.14/4.4.4_prov_ptables.md

```diff
@@ -3,6 +3,15 @@ Partition templates are a subset of normal Provisioning Templates. They are hand
 
 Foreman comes with pre-created templates for common Operating Systems, but it is good to edit the template, check it's content and it's Family setting. If the Family is wrong, be sure to go back to Operating Systems and associate it with your Operating System.
 
+#### Per-Host Partition tables
+
+When creating a new Host, you will be given the option to create an individual partition table. This is essentially a 'one-off' partition table that is stored with the host and used only for that host. It replaces the choice of Partition Table from the normal list of those associated with the selected OS.
+
 #### Dynamic Partition tables
+Some operating systems allow you to create partition tables via scripts. At the moment Kickstart and AutoYaST based systems can use this feature. Partition templates starting with #Dynamic are interpreted as scripts rather than static partition tables. The Provisioning Template needs to support this feature (search for @dynamic). This enables you to make choices on the fly during provisioning (or re-provisioning).
+
+##### Kickstart Dynamic Partition tables
+Kickstart will run dynamic partition tables as a pre-install bash script. This script needs to create a complete Kickstart partition table in '/tmp/diskpart.cfg'. This partition table will then be read by anaconda for the installation.
```

Comment:
1. "as a pre-install bash script using a %pre scriplet."
1. ".. read by anaconda for the installation by using `%include /tmp/diskpart.cfg`." (to show it's not automatic)